app/src/main/java/com/example/arc_exapmle/AddNoteKtActivity.kt

```diff
@@ -61,9 +62,10 @@ class AddNoteKtActivity : AppCompatActivity() {
 
         val note = NoteEntity(title, description, priority)
 
+
         val data = Intent()
 
-        data.putExtra(EXTRA_NOTE, note)
+        data.putExtra(EXTRA_NOTE, NoteUI(note.getId(), note.title, note.description, note.priority))
```

Comment:
va chera in payin tabdilesh mikoni be ui?
chera mostaghim nemizari to UI?

Reply:
bekhater hamon masaleye comment ghabli